### DISCUSSION TITLE
"fix for very long server names" gone?

### DISCUSSION THREAD

**Discussion by sambernet** [nginx-proxy/nginx-proxy#1913] [Q&A] [answered]
https://github.com/nginx-proxy/nginx-proxy/blob/fb8ddfd08c0f6cb53e583ea22ff8be06f69c50dc/Dockerfile#L58

I just stumpled upon the comment on Line 58 linked above while updating the nginx image for our own version of the "jwilder-proxy" (containing some customizations) to all the goodness that was added here since my last sync.

While the comment still suggests the "fix for very long server names" is applied, I don't actually *see* it being applied in the run block:

```
RUN echo "daemon off;" >> /etc/nginx/nginx.conf \
   && sed -i 's/worker_processes  1/worker_processes  auto/' /etc/nginx/nginx.conf \
   && sed -i 's/worker_connections  1024/worker_connections  10240/' /etc/nginx/nginx.conf \
   && mkdir -p '/etc/nginx/dhparam'
```

Within the nginx default config this just adapts `worker_processes` and `worker_connections` (and switches off `daemon` mode for the combined nginx-dockergen image version).

The actual, previous fix involved setting the `server_names_hash_bucket_size`:

```
sed -i 's/^http {/&\n    server_names_hash_bucket_size 128;/g' /etc/nginx/nginx.conf
```

Now I'm confused: 
*Is the fix not necessary anymore and the comment is outdated, or was the fix inadvertently removed?* 🤔 

Regards,
Sam

**Comment by buchdag** [comment, marked as answer]
Hi. The fix is now directly in the nginx.tmpl file since #589 (more specifically b0de80d46b63f353a198e2110aa2b03ad26e691d)

https://github.com/nginx-proxy/nginx-proxy/blob/fb8ddfd08c0f6cb53e583ea22ff8be06f69c50dc/nginx.tmpl#L171-L172

The comment in the Dockerfile is misleading and will be removed.

**Reply by sambernet** [reply]
Hi @buchdag 

I was trying to figure this out with `git blame` going through the `Dockerfile` but failed blatantly at identifying the correct commit. Thanks!
